diff --git a/moz.build b/moz.build
index 9db27fd056f9a1ae0b70ca0c6730574946989edf..06f114eb5d3312764e258430acae111402308aeb 100644
--- a/moz.build
+++ b/moz.build
@@ -220,3 +220,7 @@ SPHINX_TREES["mots"] = "docs/mots"
 SPHINX_TREES["update-infrastructure"] = "docs/update-infrastructure"
 
 include("build/templates.mozbuild")
+
+FINAL_TARGET_FILES.distribution += [
+    'distribution/policies.json',
+]
